OpenAI_API/Engine/Engine.cs

```diff
@@ -40,8 +40,9 @@ public class Engine
 		/// Represents an Engine with the given id/<see cref="EngineName"/>
 		/// </summary>
 		/// <param name="name">The id/<see cref="EngineName"/> to use.
-		///						If the <paramref name="name"/> contains a colon (as is the case in the API's <see cref="CompletionResult.Model"/> response),
-		///						the part before the colon is treated as the id/<see cref="EngineName"/> and the following portion is considered the <see cref="ModelRevision"/>
+		///	If the <paramref name="name"/> contains a colon (as is the case in the API's <see cref="CompletionResult.Model"/> response),
+		///	the part before the colon is treated as the id/<see cref="EngineName"/> and the following portion is considered the<see cref="ModelRevision"/>
+		///	the part before the colon is treated as the id/<see cref="EngineName"/> and the following portion is considered the <see cref="ModelRevision"/>
```

Comment:
I am on mobile at the moment so it's possible this is a formatting glitch, but it appears this line is duplicated￼